src/Workspaces/Core/Portable/Options/NamingEditorConfigStorageLocation.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    internal sealed class NamingEditorConfigStorageLocation : OptionStorageLocation { }
```

Comment:
You might want to comment why this type is there. I don't disagree with it's existence (and now want to refactor the language settings one for clarity too...) but a comment is good.